noise_raster/raster_processing.py

```diff
@@ -159,15 +194,13 @@ def create_zero_array(in_ds):
 
     return zeroData
 
-    ds = None
-
-def create_raster(sound_array:np.ndarray, merged_vrt):
+def create_raster(sound_array:np.ndarray, merged_vrt, out_pth=None):
```

Comment:
General comment on variable naming (not a blocker for merging):

Understandability of variable names is generally more important than saving characters to shorten them. `out_pth` vs. `out_path` saves only a single character but makes the name harder to understand. Anecdotally, the first time I saw it, I was wondering the _p_th element of something was meant here.

Reply:
OK, I wouldn't have assumed that. This one is clear to me and I have seen it used in many code examples, however it is good to know that this is not clear for you.